app/helpers/application_helper.rb

```diff
@@ -18,10 +18,10 @@ def view_class
   end
 
   def title(page_title)
-    derived_title = if page_title.include?(ApplicationConfig["COMMUNITY_NAME"])
+    derived_title = if page_title.include?(community_name)
                       page_title
                     else
-                      page_title + " - #{ApplicationConfig['COMMUNITY_NAME']} Community 👩‍💻👨‍💻"
+                      page_title + " - #{community_qualified_name} 👩‍💻👨‍💻"
```

Comment:
I think it makes sense to replace with ` "#{page_title} - #{community_qualified_name} 👩‍💻👨‍💻"` as long with the current change for consistency, and following [the style guide](https://github.com/rubocop-hq/ruby-style-guide#string-interpolation).

Reply:
Makes sense :) Done!